.bit/tests/sample-solutions/week3/3.4-bunnimage.js

```diff
@@ -1,40 +1,25 @@
 const { BlobServiceClient } = require("@azure/storage-blob");
-const connectionstring = process.env["AZURE_STORAGE_CONNECTION_STRING"];
-const account = "bunnimagestorage";
 
-// https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-nodejs
-// 0 */5 * * * *
-// https://crontab.guru/every-5-minutes
+/* 
+Timer-triggered Azure Function need a storage account
+When developing locally do not forget to set the "AzureWebJobsStorage": "UseDevelopmentStorage=true"
+in the local.settings.json file
+*/
+
+//Cron values (see function.json - schedule)
+//0 */5 * * * *
+//https://crontab.guru/every-5-minutes
 
 module.exports = async function (context, myTimer) {
-    // create blobserviceclient object that is used to create container client
-    const blobServiceClient = await BlobServiceClient.fromConnectionString(connectionstring);
-    const deletecontainer = "images";
-    const deletecontainerClient = await blobServiceClient.getContainerClient(deletecontainer);    
-    for await (const blob of deletecontainerClient.listBlobsFlat()) {
-        context.log('\t', blob.name);
-        await deleteBlob(blob.name)
-    }
-    context.log("Just deleted your blobs!")
 
-};
+    const connectionString = process.env.AZURE_STORAGE_CONNECTION_STRING;
+    const deleteContainerName = "bunnimage-upload";
```

Comment:
i think in the directions we called this container `images`, should we have it called that here as well?

Reply:
Good catch - Is now part of PR

Reply:
Looks great!